### PR TITLE
Facade OOP Design Pattern

### PR DESCRIPTION
**Pull request by tonilopezmr**
Solved #14 , #16 

🔥🔥 **ALSO REPACKAGE PROJECT IN Object Oriented Programming and Functional Programming** 🔥🔥

It's very easy pattern only a Object abstraction that do operations.

In this case calculate a net salary using IVA and IRPF operations.

### CODE REVIEW

File: src/main/kotlin/oop/Facade/Facade.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  fun calculate(salary: Double) : Double =
+    salary + iva.apply(salary) - irpf.apply(salary)
+
+  fun calculateAnnual(monthlySalary: Double) : Double =
+    calculate(monthlySalary * 12)
+
+}
+
+class IVAOperation {
```
Comment (Cotel) [inline src/main/kotlin/oop/Facade/Facade.kt]: Has `IVAOperation` anything in common with `IVAStrategy` or are they different things?

Comment (tonilopezmr) [inline src/main/kotlin/oop/Facade/Facade.kt]: Different things, this is only a example of Facade.

Has not relationship with little thai

**Comment by Cotel** [comment]
Do we wait for other people to comment or we merge already?

**Comment by tonilopezmr** [comment]
I think anything can comment in any merged PR for any question o suggestion 👍

**Comment by Cotel** [comment]
Anyone*

**Pull request merged**